packages/core/src/build/service.ts

```diff
@@ -168,10 +170,22 @@ export class BuildService extends Emittery<BuildServiceEvents> {
 
     const config = result.exports.default as Config;
 
-    // TODO: Validate config lol
+    try {
+      await validateConfig({ config });
```

Comment:
Up to you, but I might consider having this function return `Error | undefined` rather than throw.